Dockerfile

```
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi9/go-toolset:1.19 as builder
```

should we change the tag to use 1.20 to match the go.mod go version? 
```suggestion
FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi9/go-toolset:1.20 as builder
```